README.md

```diff
@@ -30,32 +30,34 @@ locally and review your changes, you will need to do the following:
 2. Install Julia by following [the relevant
    guide](https://julialang.org/downloads/).
 
-3. Navigate in a terminal to `Turing.jl/docs/`. Run `julia make.jl`,
+3. Navigate in a terminal to `turing.ml/`. Run `julia make_local.jl`,
    then you'll get a `_docs` folder full of documents. A `_tutorials`
    folder is also copied from the Turing Tutorial Repo. You might need
    to install `Documenter` and `DocumenterMarkdown` first. You can
    install these packages by running `julia -e 'using
    Pkg;Pkg.add("Documenter");Pkg.add("DocumenterMarkdown");Pkg.add("DynamicHMC");Pkg.add("Turing");Pkg.build()'`.
+   You might need to install additional packages. Just follow the same instructions as above to install any missing
+   packages.
 
 4. Type `bundle exec jekyll serve` if you wish to review the changes
    in a browser. Typically the website will be served on
    `localhost:4000`, which you can visit in your browser.
 
-6. Alternatively, if you simply want to build the site files into a
+5. Alternatively, if you simply want to build the site files into a
    static site, you can use `bundle exec jekyll build`.
 
 The files in `_docs/` may be out of date with the website. If this is
-the case, execute the `make.jl` script found at
-`Turing.jl/docs/make.jl`, which will run all the documents in the
-`docs/src` folder through a markdown processor and place them into the
-`docs/_docs/` folder. Any files located in the `docs/_docs/` or
-`docs/_tutorials/` directories should **not** be edited directly, as
-they will be overwritten by the versions in the `docs/src/`
-directory. Edits must be made to the `docs/src/` versions.
+the case, execute the `make_local.jl` script, which will run all the documents in the
```

Comment:
Could we use the make.jl script instead? It seems a bit annoying to keep two somewhat reeundant scripts updated and easier to just work with a single script.

Reply:
I completely agree with that, however, I was not sure if changing the `make.jl` file would mess with deploying the live website. If someone knows that it is not the case, there should only be one file for sure.

Reply:
If I understand the website build correctly, then the website is built into a temporary directory and then deployed from that directory? We could build a separate local version in the same file, which is then not deployed, but accessed locally. Does that make sense?

Reply:
Yes, that sounds sensible.